docs/examples.rst

```diff
@@ -1,9 +1,27 @@
 
 Examples
---------
+=========
 
-Explain any pre-reqs needed by users in order to run the examples below
+Pre Requisites
+---------------
 
+Before we run the below examples, there are a few things we need to have up and running. 
+
+GlobusConnectPersonal
+######################
+
+To run tools like ``gladier_tools.globus.Transfer``, we need access to Globus endpoints. One can use tutorial endpoints 1 and 2 `here <https://app.globus.org/file-manager>`_.
```

Comment:
Might be good to list the main endpoint page for each endpoint listed

https://app.globus.org/file-manager/collections/ddb59aef-6d04-11e5-ba46-22000b92c6ec/overview?back=endpoints